packages/core/contracts-ovm/insured-bridge/interfaces/BridgeAdminInterface.sol

```diff
@@ -14,6 +24,8 @@ interface BridgeAdminInterface {
         address bridgePool;
     }
 
+    function whitelistedTokens(address) external view returns (L1TokenRelationships memory);
```

Comment:
Adding this function + the events above enables the off-chain clients to use the `BridgeAdminInterface` instead of having to specify `OptimismBridgeAdmin` vs `ArbitrumBridgeAdmin`.